### DISCUSSION TITLE
basic question

### DISCUSSION THREAD

**Discussion by skyw33** [N4S4/synology-api#177] [General]
This is probably  a stupid question, but is there any way to access the synology api from within an already authenticated ssh session?  In other words, if I'm already logged in, I theoretically wouldn't need to login via http request.  I'm not a developer, so go easy on me.  Thanks.

**Comment by N4S4** [comment]
Hi, no worries, unfortunately the only way to get a sid or token is to input parameters. The repo is not made to interact with synology package from same machine
